R/build.r

```diff
@@ -28,6 +29,10 @@
 #' Supplying this will:
 #' * Allow other pkgdown sites to link to your site when needed,
 #'   rather than using generic links to <https://rdrr.io>.
+#'   * This is done by hosting machine readable metadata about your site at
```

Comment:
I think it would be better to just link to the auto-linking vignette here; otherwise I think the details are distracting.

Reply:
forgot about the auto-linking vignette. 

Perhaps the information about `deploy` should also be moved there?

Reply:
resolved by 334c538